src/earn/poolInfoScreen/WithdrawBottomSheet.tsx

```diff
@@ -150,7 +150,9 @@ export default function WithdrawBottomSheet({
     >
       <View style={styles.actionsContainer}>
         <PartialWithdrawAction forwardedRef={forwardedRef} pool={pool} hasRewards={canClaim} />
-        {canClaim && <ClaimAction forwardedRef={forwardedRef} pool={pool} />}
+        {canClaim && !pool.dataProps.withdrawalIncludesClaim && (
```

Comment:
@finnian0826 made this change to prevent claim action from being displayed when `withdrawalIncludesClaim` is true, the result is the same for each.

Reply:
we still want to show ClaimAction even when `withdrawalIncludesClaim` is true, because users can still claim their earnings without withdrawing.

Reply:
Reverted this change in 01ec110263c01a9934a79060a2659d3df89cfb64.